src/backend/marsha/core/views.py

```diff
@@ -85,7 +85,7 @@ def get_context_data(self):
                 "resource_data": "null",
             }
 
-        context = {"state": INSTRUCTOR if lti.is_instructor else STUDENT}
+        context = {"state": STUDENT if lti.is_student else INSTRUCTOR}
```

Comment:
Should the state be replaced by "role" and pass the real role and let the frontend decide how to treat them?

If not can we find another binary name for the state e.g view / edit ?

Reply:
the state is also used to knoq if there is an error.

Reply:
The front application accepts 2 values: `error`, `intructor`, `student`